frontend/src/App.vue

```diff
@@ -1,23 +1,28 @@
 <template>
-  <div id="app" class="d-flex flex-column yeoman-ui vld-parent">
-    <loading :active.sync="isLoading"
+  <v-app id="app">
+        <loading :active.sync="isLoading"
              :is-full-page="true"
              :height="128"
              :width="128"
              :color="isLoadingColor"
              loader="dots">
     </loading>
+    <div v-if="!prompts.length" class="loading">{{ messages.generators_loading }}</div>
+
     <Header
       v-if="prompts.length"
       :selectedGeneratorHeader="selectedGeneratorHeader"
       :stepName="prompts[promptIndex].name"
       :rpc="rpc"
+      @parentShowConsole="toggleConsole"
     />
-    <v-row class="m-0 p-0">
-      <v-col class="navigaiton-row m-0 p-0" lg="3" sm="auto">
-          <Navigation v-if="prompts.length > 0" :promptIndex="promptIndex" :prompts="prompts" />
+    <v-row class="main-row m-0 p-0">
+      <v-col class="left-col m-0 p-0" lg="3" sm="auto">
+        <Navigation v-if="prompts.length" :promptIndex="promptIndex" :prompts="prompts" />
```

Comment:
why did you revert to 'v-if="prompts.length"' ?

Reply:
Done